pkg/controller/vsphere/machine_scope_test.go

```diff
@@ -276,11 +282,34 @@ func TestGetCredentialsSecret(t *testing.T) {
 }
 
 func TestPatchMachine(t *testing.T) {
+	g := NewWithT(t)
+
+	configv1.AddToScheme(scheme.Scheme)
+
+	testEnv := &envtest.Environment{
+		CRDDirectoryPaths: []string{filepath.Join("..", "..", "..", "config", "crds")},
+	}
+
+	cfg, err := testEnv.Start()
+	g.Expect(err).ToNot(HaveOccurred())
+	g.Expect(cfg).ToNot(BeNil())
+	defer func() {
+		g.Expect(testEnv.Stop()).To(Succeed())
+	}()
+
+	k8sClient, err := client.New(cfg, client.Options{})
```

Comment:
You'll need to pass `Scheme: scheme.Scheme` here to make the client aware of the APIs

Reply:
Does this need addressing?

Reply:
No, client.New() will use default scheme